tools/lur-command.man

```diff
@@ -0,0 +1,49 @@
+.TH lur\-command 1 "Oct 26, 2016" lur\-command
```

Comment:
pls make this the same as the one from ratbag-command (adjust that one in a separate commit, where neceessary).

Specifically: instead of a hardcoded date please use @version@ and hook it into the build system so that gets handled correctly.